docs/samples/Microsoft.ML.Samples/Dynamic/BinaryClassification/AveragedPerceptron.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Microsoft.ML;
+
+namespace Microsoft.ML.Samples.Dynamic.BinaryClassification
+{
+    public static class AveragedPerceptron
```

Comment:
>AveragedPerceptron [](start = 24, length = 18)

Can I convince you to move this file to Dynaic/Trainers/BinaryClassification?

Reply:
+1

---
In reply to: [255262779](https://github.com/dotnet/machinelearning/pull/2483#discussion_r255262779) [](ancestors = 255262779)

Reply:
done

---
In reply to: [255266020](https://github.com/dotnet/machinelearning/pull/2483#discussion_r255266020) [](ancestors = 255266020,255262779)